apps/extension/src/state/tx-approval.ts

```diff
@@ -1,13 +1,10 @@
 import { AllSlices, SliceCreator } from './index';
-import { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1alpha1/custody_pb';
-import { Jsonified } from '@penumbra-zone/types';
 import { MessageResponder } from '@penumbra-zone/types/src/internal-msg/shared';
-import { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
 import { TxApproval } from '@penumbra-zone/types/src/internal-msg/tx-approval';
 
 export interface TxApprovalSlice {
-  authorizeRequest?: Jsonified<AuthorizeRequest>;
-  transactionViewFromPlan?: Jsonified<TransactionView>;
+  authorizeRequest?: string;
+  transactionViewFromPlan?: string;
```

Comment:
Why are json strings better than Json objects? Something like `Jsonified<AuthorizeRequest>` feels much more expressive here.

Reply:
it's an alias to jsonobject and zustand refuses to store jsonobject

Reply:
wrapped with a `Stringified<J>` type that aliases to string